website/docusaurus.config.js

```diff
@@ -177,7 +177,7 @@ const config = {
       prism: {
         theme: themes.github,
         darkTheme: themes.dracula,
-        additionalLanguages: ["go", "sql", "json", "json5", "systemd"],
+        additionalLanguages: ["go", "sql", "json", "systemd"],
```

Comment:
Let's figure out how that setting works in a separate PR.

I discovered that the set of standard languages changes over time. So I tried to add all languages we actually use in #4847 (and removed/repalced a few that we don't use). Somehow, that completely broke aliases: `js` stopped working, `javascript` became required. Could you check this out?